src/api/Config/types.ts

```diff
@@ -45,10 +45,28 @@ export interface Configuration {
   plugins?: IPluginConfig;
   dataSource?: SelectedDataSourceConfig;
   needRestart?: boolean;
+  emailNotifications: EmailNotifications;
```

Comment:
@maxime-allex If you scroll down to the bottom, we already have an interface for this called `IEmailNotificationsConfig`

Reply:
ok